assets/stylesheets/modules/_checkout.scss

```diff
@@ -209,6 +209,11 @@ $sprite-card-offset-visa:               -355px;
         margin-left: 35%;
     }
 }
+.review-details__notes {
```

Comment:
Looking at Ben's designs, he wants the same styling on the text on the right-hand side, so scoping this to a child of review-details might not be the best decision. You may want to deal with that in the PR for the right-hand side text though